docs/api-markdown-documenter/alert-node.js

```diff
@@ -1,44 +0,0 @@
-/*!
```

Comment:
`api-markdown-documenter` infra now lives under `infra/api-markdown-documenter`. The contents changed sufficiently enough that Git does not treat it as a refactor.